controllers/configmap_controller.go

```diff
@@ -67,9 +67,16 @@ const (
 	ConfigMapController = "configmap"
 )
 
+// newServiceConfig returns a 'Data' object which contains the expected Windows service configuration based on
+// the state of the cluster
+func newServiceConfig() (*servicescm.Data, error) {
```

Comment:
I don't understand the  need for this wrapper. Also shouldn't it be `newServicesConfig()`?

Reply:
I'm jumping the gun adding this function, removing it for now.
There will be a need for it as the logic for the service definitions grows, but its not necessary now.